platform/src/Playground.js

```diff
@@ -262,8 +263,8 @@ function initialisePanels() {
    /**
      * Create a panel for a given panel config entry
      * 
-     * @param {string} panel 
-     * @return {Panel}
+     * @param {Object} activity config panel definition 
+     * @return {Panel} platform Panel
      */
     function createPanelForDefinitionId(panel){
```

Comment:
Is the parameter name `panel` still correct?

Reply:
The parameter name is as intended the updated description removed the parameter name.
I'll update it to `@param {Object} panel -  The activity config panel definition.`

The jsdoc comment style supports dashes to separate paramter name from description which make this clearer raised #107 to apply consistently.